cmd/loop/loopout.go

```diff
@@ -25,10 +27,10 @@ var loopOutCommand = cli.Command{
 	Optionally a BASE58/bech32 encoded bitcoin destination address may be
 	specified. If not specified, a new wallet address will be generated.`,
 	Flags: []cli.Flag{
-		cli.Uint64Flag{
+		cli.StringFlag{
 			Name: "channel",
-			Usage: "the 8-byte compact channel ID of the channel " +
-				"to loop out",
+			Usage: "the comma-separated list of 8-byte compact " +
```

Comment:
Is "compact channel ID" a term that end users are familiar with?

Reply:
replaced by 'short' which I think is mostly used